html/index.html

```diff
@@ -7,11 +7,35 @@
   <link rel="stylesheet" href="https://ebi.emblstatic.net/web_guidelines/EBI-Icon-fonts/v1.2/fonts.css" type="text/css" media="all" />
 
   <link rel="stylesheet" href="http://localhost:8080/gxa/sc/resources/css/theme-atlas.css" type="text/css" media="all">
+
+  <link rel="stylesheet" type="text/css" charset="UTF-8" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" />
+  <link rel="stylesheet" type="text/css" charset="UTF-8" href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css" />
+  <style type="text/css">
```

Comment:
Is there a way we can add these classes as CSS-in-JS in the carousel component?

Reply:
Sorry I did not find a way to make this. May I have any hint?

Reply:
I was asking since in [the docs](https://react-slick.neostack.com/docs/get-started) it says that:
> Note: But be aware slick-carousel has a peer-dependancy on jQuery which you, or your colleagues may not like to see in your console output, so you can always grab the CSS from there and convert it into any CSS in JS solution that you might be using.

Have a look at [`createGlobalStyle` in styled-components](https://www.styled-components.com/docs/api#createglobalstyle). It’s not the most elegant solution but it’s useful.